tools/init.sh

```diff
@@ -6,6 +6,6 @@ CURRENT_DIR=$(cd $(dirname $0); pwd)
 source ${CURRENT_DIR}/env
 
 ${CURRENT_DIR}/init_cli.sh
+${CURRENT_DIR}/enable_service.sh
```

Comment:
can you change script name more generic.
e.g. init_cloudprovider.sh, prepare_cloud.sh and so on.

Reply:
OK, done.